### PR TITLE
Fixes some more goddamn typos and bad writing.

### PR DESCRIPTION
**Pull request by harmonyn**
no I have nothing better to do

### CODE REVIEW

File: code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm [hunk 1]
```diff
@@ -162,7 +162,7 @@
 	channel_time = 80
 	consumed_components = list(BELLIGERENT_EYE = 1, VANGUARD_COGWHEEL = 1, REPLICANT_ALLOY = 2)
 	object_path = /obj/structure/destructible/clockwork/shell/fragment
-	creator_message = "<span class='brass'>You form an anima fragment, a powerful soul vessel receptable.</span>"
+	creator_message = "<span class='brass'>You form an anima fragment, a powerful soul vessel receptacle.</span>"
```
Comment (kevinz000) [inline code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm]: recep table

File: code/game/gamemodes/cult/ritual.dm [hunk 2]
```diff
@@ -28,7 +28,7 @@ This file contains the arcane tome files.
 		return ..()
 	if(iscultist(M))
 		if(M.reagents && M.reagents.has_reagent("holywater")) //allows cultists to be rescued from the clutches of ordained religion
-			user << "<span class='cult'>You remove the taint from [M].</span>"
+			user << "<span class='cult'>You remove the taint from [M].</span>" // fucking ow
```
Comment (lzimann) [inline code/game/gamemodes/cult/ritual.dm]: ??

Comment (XDTM) [inline code/game/gamemodes/cult/ritual.dm]: i think removing the taint would be pretty painful

Comment (ExcessiveUseOfCobblestone) [inline code/game/gamemodes/cult/ritual.dm]: why is this so funny

Comment (SweaterKittens) [inline code/game/gamemodes/cult/ritual.dm]: I fucking died reading this part, haha

**Pull request merged**